src/Organizer/Commands/UpdateGeoCoordinates.php

```diff
@@ -0,0 +1,40 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Organizer\Commands;
+
+use CultuurNet\UDB3\Address\Address;
+
+class UpdateGeoCoordinates
```

Comment:
Nit: If I remember correctly we called it `UpdateGeoCoordinatesFromAddress` for `Offer`s, in case we'd ever want to add a way to manually set the coordinates. Because then it would make more sense to have `UpdateGeoCoordinates` for id + coordinates and `UpdateGeoCoordinatesFromAddress` for id + address. But commands are not persisted so we could always change it at that point.